sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/OpenAIClientTestBase.java

```diff
@@ -112,7 +123,9 @@ OpenAIClientBuilder getNonAzureOpenAIClientBuilder(HttpClient httpClient) {
     private void addTestRecordCustomSanitizers() {
         interceptorManager.addSanitizers(Arrays.asList(
             new TestProxySanitizer("$..key", null, "REDACTED", TestProxySanitizerType.BODY_KEY),
-            new TestProxySanitizer("$..endpoint", null, "https://REDACTED", TestProxySanitizerType.BODY_KEY)
+            new TestProxySanitizer("$..endpoint", null, "https://REDACTED", TestProxySanitizerType.BODY_KEY),
+            new TestProxySanitizer("Content-Type", "(^multipart\\/form-data; boundary=[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{2})",
```

Comment:
Could a body sanitizer be used to match on the boundary format (`--[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{2}`) and sanitize it instead of using bodiless matching? Also, minor nit could we change the redaction value to `BOUNDARY` as we don't need to redact it but standardize it.

Reply:
It doesn't work in the request body somehow, I tried with exact same regexp 
` new TestProxySanitizer( "(--[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{2})", "REDACT". TestProxySanitizerType.BODY_REGEX)`

Reply:
Since the request body is form data, you can use RecordWithoutRequestBody